test/transmit.js

```diff
@@ -428,7 +428,7 @@ describe('transmission', () => {
             const log = server.events.once('response');
             const client = Net.connect(server.info.port, () => {
 
-                client.write('GET / HTTP/1.1\r\naccept-encoding: gzip\r\n\r\n');
+                client.write('GET / HTTP/1.1\r\nHost: host\r\naccept-encoding: gzip\r\n\r\n');
```

Comment:
Why did you add the `Host` header?

Reply:
Because it is required to be a valid HTTP/1.1 request, and the node v20+ parser is more pedantic, no longer generating requests from this.

Reply:
Ok, thanks.